src/api.v2/helpers/worker/generateEtag.js

```diff
@@ -0,0 +1,39 @@
+const hash = require('object-hash');
+const config = require('../../../config');
+const getExperimentBackendStatus = require('../backendStatus/getExperimentBackendStatus');
+const getExtraDependencies = require('./workSubmit/getExtraDependencies');
+
+const createObjectHash = (object) => hash.MD5(object);
+
+const generateETag = async (
+  data,
+) => {
+  const {
+    experimentId,
+    body,
+    extras,
+    cacheUniquenessKey = null,
```

Comment:
This `cacheUniquenessKey` is never used, so I would say just remove it as a parameter in the function.

You could also remove it fully (from the `ETagbody` too), that would mean that all the work results in prod will be invalidated but that's not necessarily a problem, we do it from time to time.

For context, it is something that agi had added to test out things, the key needs to be set from the browser so it should either be reimplemented or removed

Reply:
I have it in my list to reimplement in the future. Running locally now, the cache will still be used which is not the case before this PR. I just didn't want to make this PR even longer

Reply:
Ok, could you make a comment here then that this param is not working currently and that there are plans to reimplement it?

Reply:
ok I just mean something explaining that this is planning to be reimplemented, because if you are leaving this here on purpose and someone runs into it and notices that it isn't being used in the future they might just remove it as a "clean up" becuase they lack all of this context